yarn.lock

```diff
@@ -16093,10 +16093,10 @@ prettier-linter-helpers@^1.0.0:
   dependencies:
     fast-diff "^1.1.2"
 
-prettier@^2.1.2, prettier@^2.7.1, prettier@^2.8.0:
-  version "2.8.1"
-  resolved "https://registry.yarnpkg.com/prettier/-/prettier-2.8.1.tgz#4e1fd11c34e2421bc1da9aea9bd8127cd0a35efc"
-  integrity sha512-lqGoSJBQNJidqCHE80vqZJHWHRFoNYsSpP9AjFhlhi9ODCJA541svILes/+/1GM3VaL/abZi7cpFzOpdR9UPKg==
+prettier@^2.1.2, prettier@^2.6.2, prettier@^2.7.1, prettier@^2.8.0:
+  version "2.8.8"
```

Comment:
Prettier bumping to 2.8.8 is the reason for all of the "unrelated" lint changes. Can you back this out?
You can still update prettier, but it should be split to a separate PR.

Reply:
I'll make a separate PR 👍

Reply:
i'll rebase once the other pr is merged